src/Components/FormPage/FormPage.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react';
+import './FormPage.css';
+import arrow from '../../images/arrow.png';
+import { postNewForm } from '../../apiCalls';
+import Question1 from './Questions/Question1';
+import Question2 from './Questions/Question2';
+import Question3 from './Questions/Question3';
+import Question4 from './Questions/Question4';
+import {QuestionComponents, TechStacks, ErrorConditions, TimeFrame, ErrorMsg, FormData} from './FormPageTypes';
+
+const FormPage = () => {
+  const [currentQuestion, setCurrentQuestion] = useState<number>(1);
+  const [stack, setStack] = useState<string>("");
+  const [searchTerm, setSearchTerm] = useState<string>("");
+  const [technologies, setTechnologies] = useState<string[]>([]);
+  const [timeframe, setTimeframe] = useState<TimeFrame>({amount: 0, type: "days"});
+  const [numPeople, setNumPeople] = useState<number>(0);
+  const [error, setError] = useState<ErrorMsg>({error: false, message: ""})
+
+  const questions: Record<number, string> = {
+    1: "Choose Your Application Type",
+    2: "Which Technologies Would You Like to Use?",
+    3: "How Long Do You Have to Create This App?",
+    4: "How Many Collaborators Will There Be?"
+  }
+
+  const techStacks: TechStacks = {
+    frontend: ['react', 'typescript', 'javascript', 'vue', 'angular'],
+    backend: ['ruby/rails', 'postgresql', 'node', 'sidekiq', 'devise'],
+    fullstack: ['react', 'typescript', 'javascript', 'vue', 'angular', 'ruby/rails', 'postgresql', 'node', 'sidekiq', 'devise']
+  }
+
+  const questionComponents: QuestionComponents = {
+    1: <Question1 chooseStack={(e) => chooseStack(e)} />,
+    2: <Question2 setTechnologies={setTechnologies} searchTerm={searchTerm} techStacks={techStacks} setError={setError} stack={stack} setSearchTerm={setSearchTerm} technologies={technologies}/>,
+    3: <Question3 setTimeframe={setTimeframe} timeframe={timeframe}/>,
+    4: <Question4 numPeople={numPeople} setNumPeople={setNumPeople}/>
+  }
+
+  const errorConditions: ErrorConditions = {
+    1: [!stack, "Please select stack!"],
+    2: [technologies.length < 1, "Please add technologies!"],
+    3: [timeframe.amount < 1, "Please select aount of time!"],
+    4: [numPeople < 1, "Please select number of collaborators!"]
+  }
+
+  const chooseStack = (e: React.MouseEvent<HTMLElement, MouseEvent>): void => {
+    const className = (e.target as HTMLInputElement).classList[0];
+    setStack(className);
+  }
+
+  const setActiveFocus = () => {
+    const filteredStacks = Object.keys(techStacks).filter(stacks => stacks !== stack);
+
+    document.querySelector(`.${stack}-div`)?.classList.add('form-active-focus');
+    filteredStacks.forEach(stacks => document.querySelector(`.${stacks}-div`)?.classList.remove('form-active-focus'));
+  }
```

Comment:
Question - could we remove the need for query selecting and iterating over these dom elements if we just put the active focus class on those elements conditionally based on whether or not stack equals that stack?